### ISSUE TITLE
SQS Offline - MalformedQueryString

### ISSUE THREAD

**Issue by esetnik** [CoorpAcademy/serverless-plugins#111]
After upgrading from v3.0.1 to v3.1.1 I get an error MalformedQueryString. It looks like the RedrivePolicy might be the problem, although this config was previously working and also matches the serverless documentation.

```
    EmailQueue:
      Type: AWS::SQS::Queue
      Properties:
        QueueName: queue-name-${self:provider.stage}
        MessageRetentionPeriod: 1209600
        RedrivePolicy:
          deadLetterTargetArn:
            'Fn::GetAtt':
              - EmailDeadLetterQueue
              - Arn
          maxReceiveCount: 5
```

Logs:
```
**Serverless: [AWS sqs 400 0.546s 0 retries] createQueue({
  QueueName: 'queue-name-local',
  Attributes: {
    MessageRetentionPeriod: '1209600',
    RedrivePolicy: '[object Object]'
  }
})

  Malformed Query String ---------------------------------

  MalformedQueryString: MalformedQueryString; see the SQS docs.
      at Request.extractError (/app/node_modules/aws-sdk/lib/protocol/query.js:50:29)
      at Request.callListeners (/app/node_modules/aws-sdk/lib/sequential_executor.js:106:20)
      at Request.emit (/app/node_modules/aws-sdk/lib/sequential_executor.js:78:10)
      at Request.emit (/app/node_modules/aws-sdk/lib/request.js:683:14)
      at Request.transition (/app/node_modules/aws-sdk/lib/request.js:22:10)
      at AcceptorStateMachine.runTo (/app/node_modules/aws-sdk/lib/state_machine.js:14:12)
      at /app/node_modules/aws-sdk/lib/state_machine.js:26:10
      at Request.<anonymous> (/app/node_modules/aws-sdk/lib/request.js:38:9)
      at Request.<anonymous> (/app/node_modules/aws-sdk/lib/request.js:685:12)
      at Request.callListeners (/app/node_modules/aws-sdk/lib/sequential_executor.js:116:18)
      at Request.emit (/app/node_modules/aws-sdk/lib/sequential_executor.js:78:10)
      at Request.emit (/app/node_modules/aws-sdk/lib/request.js:683:14)
      at Request.transition (/app/node_modules/aws-sdk/lib/request.js:22:10)
      at AcceptorStateMachine.runTo (/app/node_modules/aws-sdk/lib/state_machine.js:14:12)
      at /app/node_modules/aws-sdk/lib/state_machine.js:26:10
      at Request.<anonymous> (/app/node_modules/aws-sdk/lib/request.js:38:9)
      at Request.<anonymous> (/app/node_modules/aws-sdk/lib/request.js:685:12)
      at Request.callListeners (/app/node_modules/aws-sdk/lib/sequential_executor.js:116:18)
      at callNextListener (/app/node_modules/aws-sdk/lib/sequential_executor.js:96:12)
      at IncomingMessage.onEnd (/app/node_modules/aws-sdk/lib/event_listeners.js:307:13)
      at IncomingMessage.emit (events.js:215:7)
      at IncomingMessage.EventEmitter.emit (domain.js:475:20)
      at endReadableNT (_stream_readable.js:1184:12)
      at processTicksAndRejections (internal/process/task_queues.js:80:21)
```

**Comment by tcastelli** [comment]
Facing the same problem, the error that i get from the local java process is 
````
10:50:09.813 [elasticmq-akka.actor.default-dispatcher-10] WARN  o.e.r.s.TheSQSRestServerBuilder$$anon$1 - Cannot parse the redrive policy attribute
spray.json.JsonParser$ParsingException: Unexpected character 'o' at input index 1 (line 1, position 2), expected JSON Value:
[object Object]
 ^

        at spray.json.JsonParser.fail(JsonParser.scala:237)
        at spray.json.JsonParser.value(JsonParser.scala:79)
        at spray.json.JsonParser.values$1(JsonParser.scala:116)
        at spray.json.JsonParser.array(JsonParser.scala:120)
        at spray.json.JsonParser.value(JsonParser.scala:76)
        at spray.json.JsonParser.parseJsValue(JsonParser.scala:51)
        at spray.json.JsonParser.parseJsValue(JsonParser.scala:47)
        at spray.json.JsonParser$.apply(JsonParser.scala:30)
        at spray.json.RichString.parseJson(package.scala:50)
        at org.elasticmq.rest.sqs.CreateQueueDirectives.$anonfun$createQueue$3(CreateQueueDirectives.scala:32)
        at scala.Option.map(Option.scala:230)
        at org.elasticmq.rest.sqs.CreateQueueDirectives.$anonfun$createQueue$2(CreateQueueDirectives.scala:32)
        at akka.http.scaladsl.server.util.ApplyConverterInstances$$anon$1.$anonfun$apply$1(ApplyConverterInstances.scala:14)
        at akka.http.scaladsl.server.directives.BasicDirectives.$anonfun$tprovide$1(BasicDirectives.scala:145)
        at akka.http.scaladsl.server.Directive$$anon$1.tapply(Directive.scala:132)
        at akka.http.scaladsl.server.Directive$.$anonfun$addDirectiveApply$1(Directive.scala:144)
        at org.elasticmq.rest.sqs.CreateQueueDirectives.$anonfun$createQueue$1(CreateQueueDirectives.scala:24)
        at akka.http.scaladsl.server.Directive$.$anonfun$addByNameNullaryApply$2(Directive.scala:151)
        at akka.http.scaladsl.server.Directive$.$anonfun$Empty$1(Directive.scala:137)
        at akka.http.scaladsl.server.Directive$$anon$1.tapply(Directive.scala:132)
        at akka.http.scaladsl.server.Directive$.$anonfun$addByNameNullaryApply$1(Directive.scala:151)
        at org.elasticmq.rest.sqs.CreateQueueDirectives.createQueue(CreateQueueDirectives.scala:23)
        at org.elasticmq.rest.sqs.CreateQueueDirectives.createQueue$(CreateQueueDirectives.scala:21)
        at org.elasticmq.rest.sqs.TheSQSRestServerBuilder$$anon$1.createQueue(SQSRestServerBuilder.scala:133)
        at org.elasticmq.rest.sqs.TheSQSRestServerBuilder.rawRoutes$1(SQSRestServerBuilder.scala:180)
        at org.elasticmq.rest.sqs.TheSQSRestServerBuilder.$anonfun$start$3(SQSRestServerBuilder.scala:208)
        at org.elasticmq.rest.sqs.directives.AnyParamDirectives.$anonfun$anyParamsMap$3(AnyParamDirectives.scala:20)
        at akka.http.scaladsl.server.util.ApplyConverterInstances$$anon$1.$anonfun$apply$1(ApplyConverterInstances.scala:14)
        at akka.http.scaladsl.server.Directive.$anonfun$recover$4(Directive.scala:110)
        at akka.http.scaladsl.server.directives.BasicDirectives.$anonfun$mapRouteResult$2(BasicDirectives.scala:68)
        at akka.http.scaladsl.server.directives.FutureDirectives.$anonfun$onComplete$3(FutureDirectives.scala:37)
        at akka.http.scaladsl.util.FastFuture$.$anonfun$transformWith$1(FastFuture.scala:37)
        at akka.http.scaladsl.util.FastFuture$.strictTransform$1(FastFuture.scala:41)
        at akka.http.scaladsl.util.FastFuture$.transformWith$extension1(FastFuture.scala:45)
        at akka.http.scaladsl.util.FastFuture$.transformWith$extension0(FastFuture.scala:37)
        at akka.http.scaladsl.server.directives.FutureDirectives.$anonfun$onComplete$2(FutureDirectives.scala:37)
        at akka.http.scaladsl.server.directives.BasicDirectives.$anonfun$textract$2(BasicDirectives.scala:161)
        at akka.http.scaladsl.server.Directive.$anonfun$recover$2(Directive.scala:110)
        at akka.http.scaladsl.server.directives.BasicDirectives.$anonfun$textract$2(BasicDirectives.scala:161)
        at akka.http.scaladsl.server.directives.BasicDirectives.$anonfun$textract$2(BasicDirectives.scala:161)
        at akka.http.scaladsl.server.directives.BasicDirectives.$anonfun$mapRouteResultWith$2(BasicDirectives.scala:74)
        at akka.http.scaladsl.server.directives.BasicDirectives.$anonfun$textract$2(BasicDirectives.scala:161)
        at akka.http.scaladsl.server.directives.ExecutionDirectives.$anonfun$handleExceptions$2(ExecutionDirectives.scala:32)
        at akka.http.scaladsl.server.directives.BasicDirectives.$anonfun$mapRouteResultWith$2(BasicDirectives.scala:74)
        at akka.http.scaladsl.server.directives.BasicDirectives.$anonfun$textract$2(BasicDirectives.scala:161)
        at akka.http.scaladsl.server.directives.ExecutionDirectives.$anonfun$handleExceptions$2(ExecutionDirectives.scala:32)
        at akka.http.scaladsl.server.directives.BasicDirectives.$anonfun$textract$2(BasicDirectives.scala:161)
        at akka.http.scaladsl.server.Route$.$anonfun$asyncHandler$1(Route.scala:86)
        at akka.stream.impl.fusing.MapAsyncUnordered$$anon$31.onPush(Ops.scala:1376)
        at akka.stream.impl.fusing.GraphInterpreter.processPush(GraphInterpreter.scala:523)
        at akka.stream.impl.fusing.GraphInterpreter.execute(GraphInterpreter.scala:409)
        at akka.stream.impl.fusing.GraphInterpreterShell.runBatch(ActorGraphInterpreter.scala:606)
        at akka.stream.impl.fusing.GraphInterpreterShell$AsyncInput.execute(ActorGraphInterpreter.scala:485)
        at akka.stream.impl.fusing.GraphInterpreterShell.processEvent(ActorGraphInterpreter.scala:581)
        at akka.stream.impl.fusing.ActorGraphInterpreter.akka$stream$impl$fusing$ActorGraphInterpreter$$processEvent(ActorGraphInterpreter.scala:749)
        at akka.stream.impl.fusing.ActorGraphInterpreter$$anonfun$receive$1.applyOrElse(ActorGraphInterpreter.scala:764)
        at akka.actor.Actor.aroundReceive(Actor.scala:539)
        at akka.actor.Actor.aroundReceive$(Actor.scala:537)
        at akka.stream.impl.fusing.ActorGraphInterpreter.aroundReceive(ActorGraphInterpreter.scala:671)
        at akka.actor.ActorCell.receiveMessage(ActorCell.scala:612)
        at akka.actor.ActorCell.invoke(ActorCell.scala:581)
        at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:268)
        at akka.dispatch.Mailbox.run(Mailbox.scala:229)
        at akka.dispatch.Mailbox.exec(Mailbox.scala:241)
        at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
        at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
        at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
        at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
````

**Comment by AdrieanKhisbe** [comment]
@esetnik This is indeed an unforeseen effet of #110.

Before that the `RedrivePolicy` was not taken into account.

What could be done is either:
- Continue ignoring the `RedrivePolicy` locally
- Replace serialisation by JSON.stringify

However the later would not support your case neither as some cloudformation resolving would have to be done as the `Fn::GetAtt` is to be resolved!!

Personally so that things move quickly I will prefer to ignore the `RedrivePolicy` before to eventualy support it, along with some Cfn resolving if there is really such needs.

@esetnik, @tcastelli any thoughts?

cc @godu

**Comment by esetnik** [comment]
Yes I think it is a reasonable solution to ignore the RedrivePolicy. It’s the best short term option anyway.

**Comment by AdrieanKhisbe** [comment]
@esetnik and @tcastelli :
I Just published [serverless-offline-sqs@3.1.2](https://www.npmjs.com/package/serverless-offline-sqs/v/3.1.2)
